waspc/src/Wasp/Util/IO.hs

```diff
@@ -115,6 +116,9 @@ readFileStrict = T.IO.readFile . SP.toFilePath
 writeFile :: Path' Abs (File f) -> String -> IO ()
 writeFile = P.writeFile . SP.fromAbsFile
 
+writeFileBytes :: Path' Abs (File f) -> B.ByteString -> IO ()
+writeFileBytes = B.writeFile . SP.fromAbsFile
```

Comment:
Looking at this it looks as if it should be under sometihng like StrongPath/IO, but ok, I see other similar stuff is here, so this is consistent with it, we would have to move evertying with would now be hassle.

Reply:
Ok, I'm resolving